### ISSUE TITLE
Why can not I make a trace in a test env

### ISSUE THREAD

**Issue by rhz1949** [0xPolygonHermez/zkevm-node#2783]
I merged the deploy_sc and txsender scripts in the test/script directory to deploy a contract and then create a transaction (called "tx_A") to execute this contract. However, when I try to trace this transaction tx_A using the command:
`curl -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"debug_traceTransaction","params":["0xtx hash of tx_A", {"tracer": "calltracer"}],"id":1}' http://localhost:8123`, 

I get an error: 
`{"jsonrpc":"2.0","id":1,"error":{"code":-32601,"message":"the method debug_traceTransaction does not exist/is not available"}}.`

 How can I successfully perform a trace on this transaction?

Thank you

**Comment by obynonwane** [comment]
You need to activate debug api. Modify the command used to run the rpc.  it is in the docker compose. (same options that geth uses)

--http.api eth,net,debug,zkevm,txpool,web3

**Comment by rhz1949** [comment]
> You need to activate debug api. Modify the command used to run the rpc. it is in the docker compose. (same options that geth uses)
> 
> --http.api eth,net,debug,zkevm,txpool,web3

yes, it works after I add this command for the zkevm-json-rpc. Thank you again!

**Comment by obynonwane** [comment]
You welcome please can you share the section where you added it, in your compose file.

**Comment by rhz1949** [comment]
> You welcome please can you share the section where you added it, in your compose file.

/zkevm-node/test/docker-compose.yml (this is file path)
```
zkevm-json-rpc:
   ......// omit some code
    command:
      - "/bin/sh"
      - "-c"
      - "/app/zkevm-node run --network custom --custom-network-file /app/genesis.json --cfg /app/config.toml --components rpc --http.api eth,net,debug,zkevm,txpool,web3"
```

I add the command at the last. I don't konw if it is right, and I run the comment, it seems that this modification cannot get opcode:`{"jsonrpc":"2.0","id":1,"result":{"from":"0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266","gas":"0x5208","gasUsed":"0x5208","to":"0x959922be3caee4b8cd9a407cc3ac1c251c2007b1","input":"0x","value":"0xa","type":"CALL"}}`

maybe it's because of tx, or the node limits the use of debug API
